### ISSUE TITLE
A little suggestion

### ISSUE THREAD

**Issue by sunlewuyou** [kost/revsocks#1]
Hello, can you add the socks password verification function, so that the applicability will be wider. Thank you!

**Comment by rockyd3vil** [comment]
A little question on this should we open the ports in firewall for vps or client ?

**Comment by rockyd3vil** [comment]
Yeah as I tried this is Working like client ip using on server . but I want the server proxy on client ( which doesn't need ports to opened on firewall. because my ISP  closed all ports . can't do port forwarding . any other suggestions on this please . really appreciated.
